### PR TITLE
Updates to address changes in Patient models

### PR DESCRIPTION
**Pull request by hackrm**
The CQM-Models repository is being updated to store patient cqm and qdm data in separate models (https://github.com/projecttacoma/cqm-models/pull/69). This PR updates Bonnie to coincide with the restructuring of CQM-Models. 

**Before merging:
- [x] https://github.com/projecttacoma/cqm-converter/pull/21 must be merged first
- [x] Update the Gemfile to reference correct cqm-models and cqm-converter branches once the Measure Model work is merged in 

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1869
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [x] Automated regression test(s) pass: NA, but Excel regression was completed 

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: NA
- [x] Justification for using JIRA tests: NA
- [x] JIRA tests have been added to sprint: NA

**Reviewer 1:**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] ~JIRA tests have been run and pass~
- [x] ~You agree with the justification for use of JIRA tests or have provided input on why you disagree~

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] ~JIRA tests have been run and pass~
- [x] ~You agree with the justification for use of JIRA tests or have provided input on why you disagree~

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -6,6 +6,26 @@ GIT
     apipie-rails (0.3.6)
       json
 
+GIT
+  remote: https://github.com/projecttacoma/cqm-converter
+  revision: ce7a6110374852df22b6f4de7072ea58bcfd0168
```
Comment (lukoktonos) [inline Gemfile.lock]: This SHA is giving me an error when I try to `bundle install`:
```
$ bundle
Fetching gem metadata from https://rubygems.org/........
Fetching https://github.com/projecttacoma/cqm-converter
fatal: Could not parse object 'ce7a6110374852df22b6f4de7072ea58bcfd0168'.
Revision ce7a6110374852df22b6f4de7072ea58bcfd0168 does not exist in the repository
https://github.com/projecttacoma/cqm-converter. Maybe you misspelled it?
```

File: Gemfile [hunk 2]
```diff
@@ -30,7 +30,8 @@ gem 'cqm-converter', '~> 1.0.3'
 # gem 'bonnie_bundler', :git => 'https://github.com/projecttacoma/bonnie_bundler.git', :branch => 'master'
 # gem 'quality-measure-engine', :git => 'https://github.com/projectcypress/quality-measure-engine.git', :branch => 'master'
 # gem 'hquery-patient-api', :git => 'https://github.com/projecttacoma/patientapi.git', :branch => 'master'
-# gem 'cqm-converter', :git => 'https://github.com/projecttacoma/cqm-converter', :branch => 'master'
+gem 'cqm-converter', :git => 'https://github.com/projecttacoma/cqm-converter', :branch => 'patient_expected_values'
```
Comment (zlister) [inline Gemfile]: Just a reminder to change this before the merge

File: app/helpers/patient_helper.rb [hunk 3]
```diff
@@ -1,16 +1,15 @@
 module PatientHelper
-  
   # HDS Patient to QDM Patient model translation
```
Comment (zlister) [inline app/helpers/patient_helper.rb]: Is this getting converted to a QDM Patient Model or a CQM Patient model?

**Review by lukoktonos — DISMISSED** [review]

File: app/helpers/patient_helper.rb [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
   def self.convert_patient_models(hds_records)
-    qdm_records = []
+    cqm_records = []
     failed_records = []
 
     # Convert all of the HDS Records into QDM Records
```
Comment (zlister) [inline app/helpers/patient_helper.rb]: convert to CQM Records

File: lib/bonnie_backend_calculator.rb [hunk 5]
```diff
@@ -6,7 +6,8 @@ module BonnieBackendCalculator
 
   def self.calculate(measure, patients, value_sets_by_oid, options)
     # convert patients to QDM, note that once we switch to the QDM model this will become unnecessary (or maybe optional)
```
Comment (zlister) [inline lib/bonnie_backend_calculator.rb:8]: Not sure I understand what this comment is trying to get at. Maybe it just needs to end at `# convert patients to QDM`

File: test/unit/helpers/patient_helper_test.rb [hunk 6]
```diff
@@ -42,10 +42,10 @@ class PatientHelperTest < ActionView::TestCase
     hds_records = Record.all
     # make sure we have a total of 7 records and run converson
     assert_equal 7, hds_records.count
-    qdm_records, failed_records = PatientHelper.convert_patient_models(hds_records)
+    cqm_records, failed_records = PatientHelper.convert_patient_models(hds_records)
```
Comment (zlister) [inline test/unit/helpers/patient_helper_test.rb:45]: I can't comment on a line 24 above, but that should also be updated from `qdm_records` to `cqm_records` for consistency.

**Review by lukoktonos — DISMISSED** [review]

**Review by zlister — DISMISSED** [review]

**Review by zlister — APPROVED** [review]

**Review by lukoktonos — APPROVED** [review]

**Pull request merged**